### ISSUE TITLE
Fix timeouts

### ISSUE THREAD

**Issue by andy-maier** [andy-maier/plexmediafixup#25]
### Actual behavior

Even though the timeout is supposedly increased to 300 sec, both connections and operations time out at the requests level.

Example for connection timeout (can be triggered with direct connection in an environment that does not have a network connection to the PMS server):

```
Using plexmediafixup config file: try/ds12_config_file.yml
Using PlexAPI config file: /Users/maiera/.config/plexapi/config.ini
Loading fixup: sync_title
Loading fixup: sync_sort_title
Traceback (most recent call last):
  File "/Users/maiera/virtualenvs/plex27/bin/plexmediafixup", line 11, in <module>
    load_entry_point('plexmediafixup', 'console_scripts', 'plexmediafixup')()
  File "/Users/maiera/PycharmProjects/plex/repos/plexmediafixup/plexmediafixup/cli.py", line 281, in main
    plex = plexapi.server.PlexServer()
  File "/Users/maiera/virtualenvs/plex27/lib/python2.7/site-packages/plexapi/server.py", line 103, in __init__
    data = self.query(self.key, timeout=timeout)
  File "/Users/maiera/virtualenvs/plex27/lib/python2.7/site-packages/plexapi/server.py", line 376, in query
    response = method(url, headers=headers, timeout=timeout, **kwargs)
  File "/Users/maiera/virtualenvs/plex27/lib/python2.7/site-packages/requests/sessions.py", line 546, in get
    return self.request('GET', url, **kwargs)
  File "/Users/maiera/virtualenvs/plex27/lib/python2.7/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/maiera/virtualenvs/plex27/lib/python2.7/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/Users/maiera/virtualenvs/plex27/lib/python2.7/site-packages/requests/adapters.py", line 504, in send
    raise ConnectTimeout(e, request=request)
requests.exceptions.ConnectTimeout: HTTPConnectionPool(host='192.168.0.12', port=32400): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x1101bb190>, 'Connection to 192.168.0.12 timed out. (connect timeout=30)'))
```

### Expected behavior

timeout of 300 sec is honored for operations, but 30 sec stays for connections.

### Execution environment

* PlexMediaFixup version:
* Python version:
* Operating System (type+version):

**Comment by andy-maier** [comment]
I recently saw cases where requests times out also with a read timeout of 30sec.

**Comment by andy-maier** [comment]
Fixed in PR #39.
